addons/event_booth_sale/models/sale_order_line.py

```diff
@@ -38,11 +38,21 @@ def _compute_event_booth_pending_ids(self):
 
     def _inverse_event_booth_pending_ids(self):
         for so_line in self:
+            existing_booths = so_line.event_booth_registration_ids.event_booth_id
```

Comment:
Would add a docstring then:

```
""" This method will take care of creating the event.booth.registrations based on selected booths.
It will also unlink ones that are de-selected. """
```